server/images.go

```diff
@@ -492,6 +492,12 @@ func CreateModel(ctx context.Context, name model.Name, modelFileDir, quantizatio
 				layers = append(layers, baseLayer.Layer)
 			}
 		case "license", "template", "system":
+			if c.Name == "template" {
+				if _, err := template.Parse(c.Args); err != nil {
+					return err
```

Comment:
Possible the return a custom `ErrTemplate` here so that we can check for it specifically in `routes.go` with `errors.Is()` rather than the string prefix?

Reply:
Will do (or try to do)

Reply:
Have done